libraries/AP_OSD/AP_OSD_Screen.cpp

```diff
@@ -1923,6 +2029,154 @@ void AP_OSD_Screen::draw_esc_amps(uint8_t x, uint8_t y)
+            } else {
+                backend->write(x, y, false, "%.2f%c", value_w, SYMBOL(SYM_WATT));
+            }
+        }
+    }
+    else
+    {
+        if(btfl) {
+            backend->write(x, y, false, "---%cW", 0x0C);
+        } else {
+            backend->write(x, y, false, "---%c", SYMBOL(SYM_MW));
+        }
+    }
+}
+
+void AP_OSD_Screen::draw_crsf_rssi_dbm(uint8_t x, uint8_t y)
+{
+    const int8_t rssidbm = AP::crsf()->get_link_status().rssi_dbm;
+    const bool blink = -rssidbm < osd->warn_rssi;
+    bool btfl = is_btfl_fonts();
+
+    backend->write(x, y, blink, "%c", SYMBOL(SYM_RSSI));
+    uint8_t new_x = x + 1;
+    if (rssidbm >= 0)
+    {
+        if(btfl) {
+            backend->write(new_x, y, blink, "%4dDBM", -rssidbm);
+        } else {
+            backend->write(new_x, y, blink, "%4d%c", -rssidbm, SYMBOL(SYM_DBM));
+        }
+    }
+    else
+    {
+        if(btfl){
+            backend->write(new_x, y, blink, "----DBM");
+        } else {
+            backend->write(new_x, y, blink, "----%c", SYMBOL(SYM_DBM));
+        }
+    }
+}
+
+void AP_OSD_Screen::draw_crsf_snr(uint8_t x, uint8_t y)
+{
+    const int8_t snr = AP::crsf()->get_link_status().snr;
+    const bool blink = snr < osd->warn_snr;
+    bool btfl = is_btfl_fonts();
+    if (snr == INT8_MIN)
+    {
+        if(btfl) {
+            backend->write(x, y, blink, "SNR---DB");
+        } else {
+            backend->write(x, y, blink, "%c---%c", SYMBOL(SYM_SNR), SYMBOL(SYM_DB));
+        }
+    }
+    else {
+        if(btfl) {
+            backend->write(x, y, blink, "SNR%3dDB", snr);
+        } else {
+            backend->write(x, y, blink, "%c%3d%c", SYMBOL(SYM_SNR), snr, SYMBOL(SYM_DB));
+        }
+    }
+}
+
+void AP_OSD_Screen::draw_crsf_active_antenna(uint8_t x, uint8_t y)
+{
+    const int8_t active_antenna = AP::crsf()->get_link_status().active_antenna;
+    bool btfl = is_btfl_fonts();
+    if (active_antenna < 0) {
+        if(btfl) {
+            backend->write(x, y, false, "ANT-");
+        } else {
+            backend->write(x, y, false, "%c-", SYMBOL(SYM_ANT));
+        }
+    } 
+    else {
+        if(btfl) {
+            backend->write(x, y, false, "ANT%d", active_antenna + 1);
+        } else {
+            backend->write(x, y, false, "%c%d", SYMBOL(SYM_ANT), active_antenna + 1);
+        }
+    }
+}
+
+void AP_OSD_Screen::draw_crsf_lq(uint8_t x, uint8_t y)
+{    
+    const int16_t lqv = AP::crsf()->get_link_status().link_quality;
+    const bool blink = lqv < osd->warn_lq;
+    bool btfl = is_btfl_fonts();
+    bool prefix_rf = check_option(AP_OSD::OPTION_RF_MODE_ALONG_WITH_LQ);
+    const int16_t rf_mode = AP::crsf()->get_link_status().rf_mode;    
+    if (lqv < 0)
+    {
+        if (btfl) {
+            if(prefix_rf) {
+                backend->write(x, y, blink, "LQ--:--");
+            } else {
+                backend->write(x, y, blink, "LQ--");
+            }
+        }
+        else {
```

Comment:
```suggestion
        } else {
```

similarly everywhere else

Reply:
Ok, thanks for the head up. This was probably VSCode auto-formatting getting me off-guard. Will look into correcting this according to the style guide.